README.md

```diff
@@ -9,6 +9,7 @@ My personal blazingly fast and efficient status bar + widgets, in case anyone fi
 - Hyprland(Optional -> For workspaces widget)
 - nvidia-utils(Optional -> For Nvidia GPU status)
 - bluez(Optional -> For Bluetooth status)
+- checkupdates-with-aur (Optional -> For checking updates in Arch Linux)
```

Comment:
Should probably be ```pacman-contrib```, since this is in the official repos and used by default (gBar uses ```checkupdates``` by default, which is installed with ```pacman-contrib```). 

If you want to (no pressure though), you can add a FAQ entry on how to replace ```checkupdates``` with ```checkupdates-with-aur```